### DISCUSSION TITLE
Airflow 2.4.3 Security Risk reported in hub scan

### DISCUSSION THREAD

**Discussion by pritampbonde** [apache/airflow#28515] [General]
Hello @potiuk 

We have upgraded Airflow from 2.3.0 to 2.4.3. On performing hub scan we found out there are 1 high risk , 2 medium security risk reported mention below 

1.  no restriction on 'pig_properties'
2. OS injection
3. no restriction on some binary files

Guide us if we can raise the bug to resolve above mention issues.

Thanks,
Pritam

**Comment by potiuk** [comment]
We are publishing issues in CVE database. I recommend that in any public communication you avoid using BDSA numbers as those are proprietary. I guess if you are using Black Duck they should also explain what the remedy is - becasue we are publishing it when we sent a CVE announcement

You should find out the CVEs (which is public standard for vulnerabilties). You can find the remediation in usual public CVE lists as well you will find those at apache announcement - please subcribe to it or search throuhg public archives if you want to find out our announcements (but use CVE numbers that BDSA reports should contain - if they do not raise issue to DockeHub/BlackDuck). 

Archives are here. https://lists.apache.org/list?announce@apache.org

If you found an issue that is not public - report it following our security policy https://github.com/apache/airflow/security. certainly not via discussions and issues. 

Just to set expecations - please do not expect indivuidual responses on public vulnerabilities which we published inclduing remediations and especially - do not mark people individually. It's your job to read vulnerability reports and apply remediations that we suggested.

**Comment by pritampbonde** [comment]
Thanks you @potiuk for quick response. These CVEs are been a part of Blackduck, we are looking into the archive shared also would look for future release to get these these CVEs to be fixed.

**Reply by potiuk** [reply]
They are **likely**  fixed already (but I have no idea what the BDSE numbers are) and you need to just follow the advisories we released when we announced it with remediation - and follow those remediations. - just make sure you get to the right CVE containing the remediations.

**Reply by pritampbonde** [reply]
sure will follow the CVEs number. Hubscan stated the BDSE number which are internally mapped with generic CVEs numbers.
